components/myServices/ServiceCard.tsx

```diff
@@ -0,0 +1,64 @@
+"use client";
+
+import { Card, CardHeader } from "@nextui-org/react";
+import { useState } from "react";
+import { motion } from "framer-motion";
+
+import { serviceIcons } from "@/components/icons";
+import { services } from "@/constants";
+
+const ServiceCard = () => {
+  const [hoveredCard, setHoveredCard] = useState<number | null>(null);
+
+  const handleCardHovered = (index: number) => {
+    setHoveredCard(index);
+  };
+  const handleCardLeave = () => {
+    setHoveredCard(null);
+  };
+
+  return (
+    <>
+      {serviceIcons.map((ServiceIcon, index) => (
+        <motion.div
```

Comment:
Same thing here...

Reply:
![Screenshot 2023-09-13 at 1 11 51 AM](https://github.com/DevTaehong/portfolio_jsm/assets/71358207/85318349-292c-484f-b148-5b96368b169b)

I'm having this missing key prop error here. Did you mean this??

Reply:
I've made a reusable component called Service and mapped over it. Thanks!